src/components/FilterSearch.tsx

```diff
@@ -78,6 +78,8 @@ export interface FilterSearchProps {
   searchOnSelect?: boolean,
   /** A function which is called when a filter is selected. */
   onSelect?: (params: OnSelectParams) => void,
+  /** A function which is called when the input's value changes. This does not replace executeFilterSearch, but gets called just before it */
+  onDropdownInputChange?: (value: string) => void,
```

Comment:
I think it would be helpful to elaborate on why one would use this callback over `onSelect` which superficially sounds very similar.